### PR TITLE
Version 0.3.1

### LINKED ISSUE #8: Missing bits in Stivale2 header
the bits for fully virtual kernel mappings (3) and not failing to boot if low memory area could not be allocated (4, should always be set) don't exist in the `Header` struct

### PR DESCRIPTION
**Pull request by ominitay**
## New Features
- Added new bits to `Header.Flags` from Stivale 2 spec.

Closes #8

### CODE REVIEW

**Comment by ominitay** [comment]
@nailuj29gaming Will merge when you say you are happy with this

**Comment by nailuj29** [comment]
~~It also seems like a few tags are missing. Do you want to add those before you merge too?~~ Never mind, I'm bad at reading docs. You can merge it.

**Review by nailuj29 — APPROVED** [review]

**Comment by ominitay** [comment]
Great! LGTM

**Pull request merged**
